modules/ROOT/pages/addition/agent-installation/agent-self-signed-certificates.adoc

```diff
@@ -44,7 +44,7 @@ options (required options marked with *):
 
 If you are generating a certificate for an agent on `localhost`, `localhost` is the primary name set as Subject Name field on the certificate and is also used as the name of generated files. If `CONFIG_AGENT_NAME` is not specified, the Subject Name in agent client certificate along with the host information of agent will be used as the agent name.
 
-In order to generate a self-signed certificate for the above example, execute the following command;
+In order to generate a self-signed certificate for the above example, execute the following command:
```

Comment:
Should this file not be removed or changed to: We don't do ssc anymore please see. other page.. ?

Reply:
Okay, thanks for pointing that out. I could see how that might be confusing to our users but for now I am happy with you keeping it then